packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -124,7 +124,7 @@ export class VaFileInput {
   /**
    * Object representing a previously uploaded file. Example: `{ name: string, type: string, size: number}`
    */
-  @Prop() uploadedFile?: UploadedFile;
+  @Prop({ mutable: true }) uploadedFile?: UploadedFile;
```

Comment:
Tests were throwing warnings that the uploaded file prop was changed within the component even though the prop was set to be immutable. This change fixes that warning.